scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml

```diff
@@ -1,3 +1,54 @@
+QFN-64-1EP_9x9mm_P0.5mm_EP4.35x4.35mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.ftdichip.com/Support/Documents/DataSheets/ICs/DS_FT2232H.pdf (page 57cd)'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  #body_size_x_min: 8.9
+  body_size_x: 9.0
+  #body_size_x_max: 9.1
+  #body_size_y_min: 8.9
+  body_size_y: 9.0
+  #body_size_y_max: 9.1
+  lead_width_min: 0.18
```

Comment:
Because min/nom/max are given, please change the format to (example given, see the file for exact syntax):
```
lead_width:
  minimum: 
  nominal:
  maximum:
```